go.mod

```diff
@@ -16,38 +16,56 @@ require (
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/sync v0.2.0
 	golang.org/x/time v0.3.0
-	k8s.io/api v0.26.6
+	k8s.io/api v0.27.1
 	k8s.io/apiextensions-apiserver v0.26.6
-	k8s.io/apimachinery v0.26.6
-	k8s.io/client-go v0.26.6
-	k8s.io/component-base v0.26.6
+	k8s.io/apimachinery v0.27.1
+	k8s.io/apiserver v0.27.1
+	k8s.io/client-go v0.27.1
+	k8s.io/component-base v0.27.1
 	k8s.io/klog/v2 v2.90.1
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
+	k8s.io/kube-openapi v0.0.0-20230426210814-b0c0aaee3cc0
+	k8s.io/kubernetes v1.26.6
```

Comment:
Depending on `k8s.io/kubernetes` is not support. Which part relies on this import?